### PR TITLE
[ignore] Only for travis-ci integration testing purposes

### PR DESCRIPTION
**Pull request by pbenecki**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath("org.springframework.boot:spring-boot-gradle-plugin:1.2.4.RELEASE")
+    }
+}
+
+apply plugin: 'java'
+apply plugin: 'idea'
```
Comment (gmaslowski) [inline build.gradle]: you don't need thata

File: src/main/java/hello/Application.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.ApplicationContext;
+
+@SpringBootApplication
+public class Application {
+
+    public static void main(String[] args) {
+        ApplicationContext ctx = SpringApplication.run(Application.class, args);
+
+        System.out.println("Let's inspect the beans provided by Spring Boot:");
```
Comment (gmaslowski) [inline src/main/java/hello/Application.java]: logger
=

File: src/main/java/hello/Application.java [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+        String[] beanNames = ctx.getBeanDefinitionNames();
+        Arrays.sort(beanNames);
+        for (String beanName : beanNames) {
+            System.out.println(beanName);
```
Comment (gmaslowski) [inline src/main/java/hello/Application.java]: there's an actuator which provides you the list of created beans in the context: http://docs.spring.io/spring-boot/docs/current/reference/html/production-ready-endpoints.html

File: build.gradle (hunk 1 above)
Comment (pbenecki) [inline build.gradle]: Sure, right.
Everything here is from spring-boot "hello world" example to check travis-ci integration.
We haven't decided yet for a specific technology. Let's schedule a discussion on that :)

Comment (gmaslowski) [inline build.gradle]: A. Ok. Didn't know :P.

Comment (pbenecki) [inline build.gradle]: My fault. I didn't pass that info in its initial name.

**Pull request closed without merging**